### ISSUE TITLE
Error on front end - Uncaught Type Error

### ISSUE THREAD

**Issue by DaveBlooman** [directus/directus#1274]
For some reason, the UI has become really buggy.  I haven't changed anything and we are seeing odd behaviour.  View viewing anything, i get the error in the screenshot below.  Also, UI elements no longer work, so hitting settings cog will append #settings to the URL but not actually redirect.

## Version Info
  - Directus version and branch: 6.3.3
  - PHP version: 5.5
  - Web server: Apache
  - OS name and version: Ubuntu

## Expected Behavior

An error doesn't appear
## Actual Behavior

An error appears when navigating anywhere in the application

## Steps to Reproduce
 1. Visit UI
 2. View error

## Logs or Screenshots

<img width="292" alt="screenshot 2016-10-10 13 35 49" src="https://cloud.githubusercontent.com/assets/1492067/19236357/bb650cae-8eee-11e6-87ed-ae7bbcb7a850.png">

**Comment by WellingGuzman** [comment]
Hello @DaveBlooman Thanks for reporting this, I have two question:

1. Are you using the actual tag at 6.3.3 or an specific branch?
2. Which browser are you using?

**Comment by WellingGuzman** [comment]
This was fix here: https://github.com/directus/directus/commit/f2a06e30bcf6f53ef2b521a8943d9413dbe6651

**Comment by DaveBlooman** [comment]
I am using the 6.3.3 zip from the releases page.  It is happening in Chrome and Firefox, I haven't tried Safari.  It's odd as its just started happening today.

**Comment by WellingGuzman** [comment]
@DaveBlooman It's happening because there's a new version, and there's an error there. check the this [commit](https://github.com/directus/directus/commit/f2a06e30bcf6f53ef2b521a8943d9413dbe6651) it should fix your problem.

Update your Directus to 6.3.4

**Comment by WellingGuzman** [comment]
Let me rephrase myself, there's a new version, and we added a notification that tell user there's a new version available, so there's an error using the notification object, because it was using uppercase instead of the lowercase filename.

Hope that this solve your problem.

**Comment by DaveBlooman** [comment]
Hi, this has solved the issue
